csrc/fusion_segmenter.h

```diff
@@ -266,6 +295,11 @@ class FusionHeuristics {
   FusionHeuristics(const FusionHeuristics&) = delete;
   FusionHeuristics& operator=(const FusionHeuristics&) = delete;
 
+  SchedulerEntryOwningPtr& at(int index) {
+    NVF_ERROR(is_segmented_);
```

Comment:
nitpick: is it necessary to assert here?

For complete fusion, `heuristics_` still has one entry here right?!

Reply:
in other words, if we do intend to insert, maybe put an error message would make more sense.

Reply:
I agree that the assertion isn't necessary.